app/Mail/CheckoutComponentMail.php

```diff
@@ -26,7 +26,7 @@ public function __construct(Component $component, $checkedOutTo, User $checkedOu
         $this->note = $note;
         $this->target = $checkedOutTo;
         $this->acceptance = $acceptance;
-        $this->qty = optional($component->assets->first())->pivot->assigned_qty;
+        $this->qty = optional(optional($component->assets->first())->pivot)->assigned_qty;
```

Comment:
Why do we need two `optional`s here?

Reply:
yea, I rewrote that with ternaries instead.